sui-execution/latest/sui-move-natives/src/crypto/group_ops.rs

```diff
@@ -639,7 +647,7 @@ pub fn internal_multi_scalar_mul(
     debug_assert!(args.len() == 3);
 
     let cost = context.gas_used();
-    if !is_supported(context) {
+    if !is_msm_supported(context) {
```

Comment:
What is your take between originally invoking regular mul inside that function, instead of returning an error? This way future enablement will only make the function being executed faster in a later version. I'm happy with both options.

Reply:
I'd rather keep it simple here, and just return error